sys/include/net/ieee802154/radio.h

```diff
@@ -186,6 +190,13 @@ typedef enum {
      */
     IEEE802154_RADIO_INDICATION_RX_START,
 
+    /**
```

Comment:
I think we should add a note here hat the SFD of a transmitted ACK frame should not be reported

Reply:
we should probably extend this to the RX_START as well

Reply:
Hmm not sure about `RX_START` for a `ACK` request, ATM an `RX_DONE` is reported on an ACK being received, if we report the ACK `RX_DONE`, then the `ACK` `RX_START` should also be reported.

Reply:
Eventually we could always indicate it (if all radios support such indications). For an upper laye r(e.g SubMAC) is easy to dismiss these indications if they are not needed. But this doesn't add special constraints to layers that don't use the internal hw caps (e.g OpenWSN)

Reply:
That could be an option as well, should we move the discussion to the RDM? Seems like something it would be good to get more input on.

Reply:
@jia200x from offline discussion, the conclusion was not to report `RX_START` for ACK's but to possibly add a CAP `IEEE802154_RADIO_HAS_ACK_DONE` which should exist only when there are no ` FRAME_RETRIES`.

The only radio that doesn't support AUTOACK is the `nrf52`, in that case I would need to adapt so that the `START`/`RX_DONE` event is not reported for the `ACK`, right?

Reply:
> The only radio that doesn't support AUTOACK is the nrf52, in that case I would need to adapt so that the TX_START event is not reported for the ACK, right?

This is already the case.

Reply:
yes :)

Reply:
could you add here that the SFD of the outgoing ACK frame should not be indicated?

Reply:
Does this wording work well for you:

```
    /**
     * @brief the transceiver sent out a valid SFD
     *
     * This event is present if radio has @ref IEEE802154_CAP_IRQ_TX_START cap.
     *
     * @note The SFD of an outgoing ACK (AUTOACK) should not be indicated
     */
```

Reply:
yes, that should be enough